client-go/cmd/client/main.go

```diff
@@ -2,20 +2,43 @@ package main
 
 import (
 	"context"
+	"encoding/json"
 	"interview-client/internal/consumer"
 	"log"
+	"os"
 
 	"github.com/pkg/errors"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials/insecure"
 )
 
+var (
+	config configuration
+)
+
+type configuration struct {
+	Server string `json:"Server"`
+}
+
+func init() {
```

Comment:
Go best practices is to not use `init()` methods. Better to have a function that can be called at known points and have control of what is going on.